core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostService.scala

```diff
@@ -88,10 +88,10 @@ class SubmissionCostService(tableName: String, serviceProject: String, bigQueryD
     val queryParameters: List[QueryParameter] = List(namespaceParam)
 
     executeBigQuery(querySql, queryParameters) map { result =>
-      val rowsReturned = result.getTotalRows
-      val bytesProcessed = result.getTotalBytesProcessed
+      val rowsReturned =  Option(result.getTotalRows).getOrElse(0)
+      val bytesProcessed = Option(result.getTotalBytesProcessed).getOrElse(0)
```

Comment:
What did you look at that  told you that  `result.getTotalRows` and the next line could be null? I missed that part.

Reply:
it's in the javadoc